tests/test_read_write/test_write_jsonl.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import os
+from clumper import Clumper
+
+
+def test_local_write_exists(tmp_path):
+    """Test that an error is raised if the written file JSONL doesn't exists."""
+    path = str(tmp_path / "cards_copy.jsonl")
+    clump = Clumper.read_jsonl("tests/data/cards.jsonl")
+    clump.write_jsonl(path)
+    assert os.path.exists(path)
+
+
+def test_local_read_write_content_same(tmp_path):
+    """Test that an error is raised if the written JSONL file is not the same as what is read locally"""
+    path = str(tmp_path / "cards_copy.jsonl")
+    writer = Clumper.read_jsonl("tests/data/cards.jsonl")
+    writer.write_jsonl(path)
+    reader = Clumper.read_jsonl(path)
+    assert reader.collect() == writer.collect()
+
+
+def test_cloud_read_write_content_same(tmp_path):
+    """Test that an error is raised if the written JSONL file is not the same as what is read from the cloud"""
```

Comment:
Your docstrings follow the following pattern: 

```
Test that an error is raised if <B> does not happen. 
```

Alternatively we can gain clarity with less characters by writing. 

```
Test that B happens.
```

Reply:
Should I also refactor on the `read_json`* functions? I used the same paradigm while documenting those functions too.

Reply:
Feel free to do so, but you can also do it in a seperate PR.

Reply:
Actually, I'd be better if you did. I just had another look and found the descriptions confusing. When I read: 

```
"""Test that an error is raised if ..."""
```

Then I assume that we're testing if we raise a ValueError via `pytest.raises` which isn't what the test does.

Reply:
So keep my old paradigm? I am a bit confused now.